source/common/listener_manager/filter_chain_manager_impl.cc

```diff
@@ -285,9 +289,12 @@ void FilterChainManagerImpl::copyOrRebuildDefaultFilterChain(
       eq(origin->default_filter_chain_message_.value(), *default_filter_chain)) {
     default_filter_chain_ = origin->default_filter_chain_;
   } else {
-    default_filter_chain_ =
+    auto filter_chain_or_error =
         filter_chain_factory_builder.buildFilterChain(*default_filter_chain, context_creator);
+    RETURN_IF_NOT_OK(filter_chain_or_error.status());
```

Comment:
I think `RETURN_IF_STATUS_NOT_OK` is preferred here?

Reply:
yeah I've been using it less - added a TODO to remove entirely to avoid confusion - SG?